### DISCUSSION TITLE
Impossible access to some indicators

### DISCUSSION THREAD

**Discussion by jeaniek** [rOpenGov/eurostat#292] [General]
Hi,

I tested the updated package get_eurostat recently. I could fetch some but not all indicators, for instance, "nama_10_a64" and "nama_10_nfa_st". These latter always returns errors (Error in get_eurostat("nama_10_a64") : 
  get_eurostat_raw fails with the id nama_10_a64)
 It doesn't happen to me before I did the update to the package. Does that mean these indicators are not ready to be fetched? 

Thank you,
Nat

**Comment by pitkant** [comment]
Thank you for opening this discussion, although it may be better suited to be an issue. I tested downloading the datasets you mentioned and encountered no problems myself:

```
> nama_10_a64 <- get_eurostat("nama_10_a64")
trying URL 'https://ec.europa.eu/eurostat/api/dissemination/sdmx/2.1/data/nama_10_a64?format=TSV&compressed=true'
downloaded 17.6 MB

Table nama_10_a64 cached at /var/folders/f4/h_r3y60n0nn0qm6qx5hnx1s00000gn/T//Rtmpsg7CzX/eurostat/7bd8d32790c492cb9addb4b45f77b7be.rds

> nama_10_nfa_st <- get_eurostat("nama_10_nfa_st")
trying URL 'https://ec.europa.eu/eurostat/api/dissemination/sdmx/2.1/data/nama_10_nfa_st?format=TSV&compressed=true'
downloaded 15.2 MB

Table nama_10_nfa_st cached at /var/folders/f4/h_r3y60n0nn0qm6qx5hnx1s00000gn/T//Rtmpsg7CzX/eurostat/7d1f512f1b2ebb9d642c220c777f680c.rds
```

It might be that there's a temporary network problem, or that you're hitting a quota of some kind? These are rather big datasets, although not the biggest ones in the Eurostat database...
